src/utils/map-controls.js

```diff
@@ -86,6 +86,11 @@ export default class MapControls {
       srcEvent.ctrlKey || srcEvent.shiftKey);
   }
 
+  isRightButtonPressed(event) {
```

Comment:
Seems like something mjolnir should offer?

Reply:
Ideally. The event may be generated from one of several listeners so the format is inconsistent. We need to do an API audit on mjolnir.js.